frame/dapps-staking/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pallet-dapps-staking"
-version = "3.0.4"
+version = "3.0.5"
```

Comment:
This is a minor release bump so it should be `3.1.0`.

Please also sync the version of `pallet-precompiles-dapps-staking` with this one.
We decided to keep the versions the same since precompiles pallet has direct dependency towards `pallet-dapps-staking`.